misc/librenms-irc.service

```diff
@@ -0,0 +1,12 @@
+# /etc/systemd/system/librenms-irc.service 
+
+librenms-irc.service
+[Unit]
+Description=Librenms IRC bot
+
+[Service]
+ExecStart=/opt/librenms/irc.php
```

Comment:
Can you extract `/opt/librenms` to this:
Environment=BASE_DIR=/opt/librenms

Might be a good idea to set this to allow central configuration:
EnvironmentFile=/etc/sysconfig/librenms

But, I can do that later to unify the systemd scripts.